.github/workflows/test.yml

```diff
@@ -304,7 +304,7 @@ jobs:
       env:
         CMAKE_PARAMS: "-DVTR_ASSERT_LEVEL=3 -DWITH_BLIFEXPLORER=on"
         MATRIX_EVAL: ${{ matrix.eval }}
-        BUILD_TYPE: ${{ matrix.build }}
+        BUILD_TYPE: release
```

Comment:
We most certainly want to be building both release and debug versions? I'm unsure how we lost that?

Reply:
In several jobs, debug build types is used. These are the compatibility tests, which default to `release` only: https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/Makefile#L20. I'm not sure we want to test compatibility and debug...